js/main.js

```diff
@@ -0,0 +1,59 @@
+const hamburger = document.querySelector('.hamburger-container');
+const navbar = document.querySelector('.nav-list-items');
+
+hamburger.addEventListener(('click'), () => {
+  hamburger.classList.toggle('active');
+  navbar.classList.toggle('action');
+});
+
+// Array of project objects
+const seeMore = document.querySelector('.seemore');
+const seeInstructor = document.querySelector('.instructor-section');
+const instructorSection = document.querySelector('.instructor-container');
+
+seeMore.addEventListener(('click'), () => {
+  seeInstructor.classList.toggle('setoff');
+});
+
+const instructorContainer = [
+  {
+    name: 'Joseph Gonzalez',
+    title: 'Leading open-source projects at GitHub',
+    description: 'Joseph has been leading open-source projects at GitHub. Mostly related to JavaScript',
+    Image: 'img/joseph.jpg',
+  },
+  {
+    name: 'Jurica Koletic',
+    title: 'CEO of the marketing platform HubSpot',
+    description: 'Jurica is CEO of the marketing platform HubSpot, which he co-founded in 2006. ',
+    Image: 'img/jurica.jpg',
+  },
+  {
+    name: 'Sara Dam',
+    title: 'Fine artist-turned-coder',
+    description: 'Fine artist-turned-coder, Sara Dam took a total immersion approach to learning web development.',
+    Image: 'img/sara.jpg',
+  },
+  {
+    name: 'Ayo Ogunseinde',
+    title: 'CEO of Smartsheet',
```

Comment:
- [ ] Kindly make sure your **Instructors or Speakers** section design is much to the one from the **[Behance template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)** design for **Mobile Version** i.e by default **2 instructors** should be visible before a user clicks the `See More` button.